src/importexport/midi/internal/midishared/midifile.cpp

```diff
@@ -562,9 +562,8 @@ bool MidiFile::readEvent(MidiEvent* event)
             LOGD("readEvent: error 3");
             return false;
         }
-        data    = new unsigned char[len + 1];
```

Comment:
Maybe this needs to be replaced with `data.resize(len + 1)`? (And the same below)

I doubt whether the null termination is still necessary now that we use std::vector. So perhaps `data.resize(len)` is enough, and `data[dataLen] = 0;` can be removed.